e2e/pages/wsl-integrations-page.ts

```diff
@@ -1,13 +1,61 @@
 import { Page, Locator } from 'playwright';
+import { expect } from '@playwright/test';
+
+/**
+ * CheckboxLocator handles assertions dealing with a <Checkbox> Vue component.
+ */
+class CheckboxLocator {
+  readonly locator: Locator;
+  readonly checkbox: Locator;
+  readonly name: Locator;
+  readonly error: Locator;
+  constructor(locator: Locator) {
+    this.locator = locator;
+    this.checkbox = locator.locator('input[type="checkbox"]');
+    this.name = locator.locator('.checkbox-label');
+    this.error = locator.locator('.checkbox-outer-container-description');
+  }
+
+  click(...args: Parameters<Locator['click']>) {
+    // The checkbox itself is not visible, so it can't be clicked.
+    return this.locator.click(...args);
+  }
+
+  async assertEnabled(options?:{timeout?:number}) {
```

Comment:
What about folding these two `assertX` statements into onewith something like
```
async assertEnabled(options?:{timeout?:number}) {
  this.assertState(options, '');
}
async assertDisabled(options?:{timeout?:number}) {
  this.assertState(options, ':not');
}
```

Makes the `assertState` a touch harder to read, but then there's no duplication.

Reply:
ok